src/Nethermind/Nethermind.Core/Block.cs

```diff
@@ -120,6 +122,9 @@ public Transaction[] Transactions
     [JsonIgnore]
     public byte[][]? ExecutionRequests { get; set; }
 
+    [JsonIgnore]
+    public byte[][]? InclusionListTransactions { get; set; }
```

Comment:
We keep `Transactions` as `Transaction` object, so not sure if good idea to keep it as RLP here? Maybe decode earlier and keep it as `Transactions`?